auth-web/src/util/date-utils.ts

```diff
@@ -0,0 +1,88 @@
+import { ApiDateTimeUtc, IsoDatePacific } from '@bcrs-shared-components/interfaces'
+import isDate from 'lodash.isdate'
+
+/**
+ * A minimal class to put date-related utility methods.
+ * This contains similar methods to DateMixin in other UIs.
+ */
+export default class DateUtils {
+  /**
+   * Creates and returns a new Date object in UTC, given parameters in Pacific timezone.
+   * (This works regardless of user's local clock/timezone.)
+   * @example "2021, 0, 1, 0, 0" -> "2021-01-01T08:00:00.000Z"
+   * @example "2021, 6, 1, 0, 0" -> "2021-07-01T07:00:00.000Z"
+   */
+  static createUtcDate (year: number, month: number, day: number, hours = 0, minutes = 0): Date {
+    // 1. create the new date in UTC
+    // 2. compute the offset between UTC and Pacific timezone
+    // 3. add the offset to convert the date to Pacific timezone
+    // Ref: https://stackoverflow.com/questions/15141762/
+    const date = new Date(Date.UTC(year, month, day, hours, minutes))
```

Comment:
Use moment/or library, no point in reinventing the wheel

Reply:
Moment is pretty heavy (though it's already imported in this UI).

This is code from our "legacy" date library. JS is enough. Don't need moment 😛

Reply:
Don't need your JS library, there are professionals that do this for a living 

there are plenty of other ones